### PR TITLE
Update doc to 2.4.3

### PR DESCRIPTION
**Pull request by arnaudbore**

### CODE REVIEW

**Comment by arnaudbore** [comment]
Add section how to upgrade

File: source/pipeline/launch.rst [hunk 1]
```diff
@@ -38,24 +44,25 @@ to be executed with ``sbatch``.
...
+The following example use one node with gpu. The following lines
 must be saved in ``.sh`` file (e.g. ``cmd.sh``) to be executed with ``sbatch``.
 
 ::
 
     #!/bin/sh
 
-    #SBATCH --nodes=2
+    #SBATCH --nodes=1
+    #SBATCH --gpus-per-node=v100:1
```
Comment (AlexVCaron) [inline source/pipeline/launch.rst:58]: Remove v100 specifier `--gpus-per-node=1`

File: source/reference/changelog.rst [hunk 2]
```diff
@@ -1,6 +1,16 @@
 Changelog
 =========
 
+2.4.3
+########
+Date: October 2023
+
+New features
+  - Tractoflow is not fully working with scilpy 1.6.0
```
Comment (AlexVCaron) [inline source/reference/changelog.rst:9]: `not` ? :P

**Review by AlexVCaron — COMMENTED** [review]
Some minor comments. Once addressed, LGTM !

**Pull request merged**